deployment/docker-compose/README.md

```diff
@@ -0,0 +1,162 @@
+# Docker Compose Deployment
+
+The CCNP can be deployed in the TDVM by docker compose.
+
+## 1. Prerequisite
+
+### 1.1 Host Side 
+
+The host should enable TDX, and setup service PCCS & service QGS which help generate TDX quote in the guest.
+
+### 1.2 Guest Side
+
+The TDX device in the guest has different names in certain versions
+- `/dev/tdx-attest`: the very early name, the version is not supported. 
+- `/dev/tdx-guest` or `/dev/tdx_guest`: the name for newer version, it is supported.
+
+To enable the containers read and write the TDX device, change the access privilege of the TDX device.
+
+```
+chmod 0666 $(find /dev/ -name "tdx*")
```

Comment:
can we use udev setup as here: https://github.com/intel/confidential-cloud-native-primitives/tree/main/device-plugin/ccnp-device-plugin#prerequisite? Also I saw grant_dev_tdx seems like serve the same purpose during setup>

Reply:
Sure! And the command is for the case where the guest has not been modified by the plugin. The tool can be decoupled against outer context. 

The script `prepare.sh` is a helpful integration of all command in this section.